tests/util/model.py

```diff
@@ -86,3 +86,12 @@ def __init__(self, kernel_amplitude: float | TensorType | None = None):
 
     def __repr__(self) -> str:
         return "QuadraticMeanAndRBFKernel()"
+
+
+class LinearMeanAndRBFKernel(GaussianProcess):
+    def __init__(self, kernel_amplitude: float | TensorType | None = None):
+        kernel = tfp.math.psd_kernels.ExponentiatedQuadratic(kernel_amplitude)
+        super().__init__([linear], [kernel])
+
+    def __repr__(self) -> str:
+        return "LinearMeanAndRBFKernel()"
```

Comment:
can I suggest that, since this is only used in one place, that we just have `GaussianProcess([linear], tfp.math.psd_kernels.ExponentiatedQuadratic(kernel_amplitude))` where it's used?

Reply:
Sounds great. 
It is mainly used for test of EHVI, just would like to have a different model for different objective function, so that the predicted pareto front can be more than one point (as otherwise there might be only one dominated point).

Reply:
ok. Recall other more interesting functions are available in `trieste.utils.objectives`